rise-google-sheet.html

```diff
@@ -444,7 +457,13 @@
       ready: function() {
         var params = {
           event: "ready"
-        };
+        },
+          self = this;
+
+        // listen for data ping received
+        this.$.data.addEventListener("rise-data-ping-received", function() {
```

Comment:
@stulees In keeping with what we're trying to do in Video Widget with the whole `func-names` thing, could this be changed to `this.$.data.addEventListener("rise-data-ping-received", this._onDataPingReceived)`. We could avoid the `self = this;` as well.